# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/arch-arm.patch
# Copyright (C) 2007 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- strace-4.5.15/syscall.c.vanilla	2007-08-13 11:00:18.000000000 +0200
+++ strace-4.5.15/syscall.c	2007-08-13 11:00:44.000000000 +0200
@@ -2428,7 +2428,7 @@
 
 	switch (known_scno(tcp)) {
 #ifdef LINUX
-#if !defined (ALPHA) && !defined(MIPS) && !defined(HPPA) && !defined(AVR32)
+#if !defined (ALPHA) && !defined(MIPS) && !defined(HPPA) && !defined(AVR32) && !defined(ARM)
 	case SYS_socketcall:
 		decode_subcall(tcp, SYS_socket_subcall,
 			SYS_socket_nsubcalls, deref_style);
